packages/liveblocks-react/src/liveblocks.tsx

```diff
@@ -639,16 +635,14 @@ function useInboxNotificationsSuspense_withClient(client: OpaqueClient) {
 }
 
 function useUnreadInboxNotificationsCount_withClient(client: OpaqueClient) {
-  const { store, loadInboxNotifications, useEnableInboxNotificationsPolling } =
+  const { store, loadInboxNotifications, startPolling } =
     getExtrasForClient(client);
 
   // Trigger initial loading of inbox notifications if it hasn't started
   // already, but don't await its promise.
-  useEffect(() => {
-    loadInboxNotifications();
-  }, [loadInboxNotifications]);
+  useEffect(loadInboxNotifications, []); // eslint-disable-line react-hooks/exhaustive-deps
+  useEffect(startPolling, []); // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
Is there a particular reason why we do not add `startPolling` to the dependency here? Same for `loadInboxNotifications` above this

Reply:
We could, but I wanted to clarify those should be stable handlers that should not get executed, and the expectation is that they only execute on mount and unmount.

Reply:
But I'm also not a fan of shushing the linter, so I can add them. Technically, if the `client` instance would change you _would_ want to execute them. So maybe better anyway.